newsrevenuehub/rev-engine
# DEV-1506: bump pillow from 8.4.0 to 9.1.0

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
Bump up Pillow version to latest

#### What's this PR do?
Bumps Pillow v8.4.0 to v9.1.0

#### Why are we doing this? How does it help us?
Updates Pillow and reduces any potential risks associated with it.

#### How should this be manually tested?
Go to
`{endpoint}/edit/<{revenue program slug}`
Click on `{Edit icon}` -> Click on `{Side bar}` tab in the right panel -> Click on `{+}` icon at the bottom and add an image.
In the left panel click on `{Save}` icon.

The image details should go to table `element_media_mediaimage`.

In Django Admin Create a Donation page and add header / bg images.

#### Have automated unit tests been added?
NA

#### How should this change be communicated to end users?
NA

#### Are there any smells or added technical debt to note?
NA

#### Has this been documented? If so, where?
NA

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**
[DEV-1506]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
NA

#### Are there next steps? If so, what? Have tickets been opened for them?
NA


[DEV-1506]: https://news-revenue-hub.atlassian.net/browse/DEV-1506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
